cdc/owner/feed_state_manager.go

```diff
@@ -34,7 +43,28 @@ type feedStateManager struct {
 	// shouldBeRemoved = false means the changefeed is paused
 	shouldBeRemoved bool
 
-	adminJobQueue []*model.AdminJob
+	adminJobQueue   []*model.AdminJob
+	lastErrorTime   time.Time
+	backoffInterval time.Duration
+	expBackoff      *backoff.ExponentialBackOff
+}
+
+// newFeedStateManager creates feedStateManager and initialize the exponential backoff
+func newFeedStateManager() *feedStateManager {
+	f := new(feedStateManager)
+
+	f.expBackoff = backoff.NewExponentialBackOff()
```

Comment:
Can you extract these code into a func like 'func(m *feedStateManager) resetErrBackoff()', so that we can call it when ever we create or resume a changefeed.

Reply:
Done